### PR TITLE
feat(clients): modified grpc client constructor to accept kwargs

### PR DESCRIPTION
**Pull request by z103cb**
Modified GRPC client

Related to #29

### CODE REVIEW

File: src/caikit_nlp_client/grpc_client.py [hunk 1]
```diff
@@ -12,72 +10,25 @@
...
-
 class GrpcClient:
     """GRPC client for a caikit nlp runtime server
 
     Args:
         channel (grpc.Channel): a connected GRPC channel for use of making the calls.
     """
 
-    def __init__(self, config: GrpcConfig) -> None:
+    def __init__(self, host: str, port: int, **kwarg) -> None:
```
Comment (dtrifiro) [inline src/caikit_nlp_client/grpc_client.py:20]: Might want to explicitly add the secure/insecure argument here, pass other CA client/server certs in kwargs

Comment (z103cb) [reply src/caikit_nlp_client/grpc_client.py:20]: Let me ponder on it.

File: src/caikit_nlp_client/grpc_client.py [hunk 2]
```diff
@@ -200,3 +151,33 @@ def __populate_request(self, request, text, **kwargs):
...
+            log.debug("A CA certificate has been detected, creating a TLS channel")
+            return grpc.secure_channel(
+                connection,
+                grpc.ssl_channel_credentials(ca_cert),
+            )
+        client_key = kwargs.get("client_key")
+        client_cert = kwargs.get("client_cert")
+        server_cert = kwargs.get("server_cert")
+        if client_key and client_cert and server_cert:
+            log.debug(
```
Comment (dtrifiro) [inline src/caikit_nlp_client/grpc_client.py:174]: :100:

File: tests/test_grpc_client.py [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import pytest
-from caikit_nlp_client.grpc_client import GrpcClient
 
 
+# from caikit_nlp_client.grpc_client import GrpcClient
```
Comment (dtrifiro) [inline tests/test_grpc_client.py]: We can get rid of this

File: tests/conftest.py [hunk 4]
```diff
@@ -161,78 +162,44 @@ def channel_factory(
...
+        return grpc.secure_channel(
+            connection,
+            grpc.ssl_channel_credentials(server_cert, client_key, client_cert),
         )
-    return make_channel(config)
 
 
 @pytest.fixture(scope="session")
-def grpc_config(
+def grpc_client(
```
Comment (dtrifiro) [inline tests/conftest.py:182]: This will be pretty useful for #33

**Review by dtrifiro — APPROVED** [review]

**Pull request merged**